### ISSUE TITLE
Path concatenation error with v0.5.0-beta.4

### ISSUE THREAD

**Issue by gagle** [angular-eslint/angular-eslint#160]
I get this error:

```
An unhandled exception occurred: ENOTDIR: not a directory, stat '/Users/gllamas/projects/angular-starter/src/app/app.component.ts/0_/Users/gllamas/projects/angular-starter/src/app/app.component.ts'
```

With this config:

```
"lint": {
  "builder": "@angular-eslint/builder:lint",
  "options": {
    "eslintConfig": ".eslintrc.json",
    "fix": true,
    "lintFilePatterns": [
      "src/**/*.ts",
      "src/**/*.component.html"
    ]
  }
}
```

It is working with previous version v0.5.0-beta.3

**Comment by JamesHenry** [comment]
@gagle Please can you create a repro? Should hopefully be straightforward to use the Angular CLI workspace in the integration-tests package within this repo and create a commit which shows the issue

Thanks 🙏

**Comment by JamesHenry** [comment]
I ran into this myself today, it was a regression introduced by a recent PR, the fix is in #167

**Comment by JamesHenry** [comment]
Fix is in `v0.5.0-beta.5`
